Sources/SemanticIndex/CheckedIndex.swift

```diff
@@ -120,6 +116,13 @@ public final class CheckedIndex {
     return index.unitTests(referencedByMainFiles: mainFilePaths).filter { checker.isUpToDate($0.location) }
   }
 
+  /// Returns all the files that (transitively) include the file at the given path.
+  ///
+  /// If `crossLanguage` is set to `true`, Swift files that import a header will through a module will also be reported.
+  public func mainFilesContainingFile(path: String, crossLanguage: Bool = false) -> [String] {
+    index.mainFilesContainingFile(path: path, crossLanguage: crossLanguage)
+  }
```

Comment:
Is it expected this doesn't do any up-to-date checking?

Reply:
Ah, good call. Guess we should check.